bootstrap/src/Pharo30Bootstrap.package/PBBootstrap.class/instance/initializeImage.st

```diff
@@ -77,9 +77,7 @@ defaultAction
 	Smalltalk exitFailure
 ! !    
 
-SmalltalkImage classPool at: #SpecialSelectors put: #(#+ 1 #- 1 #< 1 #> 1 #''<='' 1 #''>='' 1 #= 1 #''~='' 1 
```

Comment:
this code was broken and is not used anymore

Reply:
Yes, we should do a pass on the bootstrap package to clean it up a bit.